index.js

```diff
@@ -31,14 +47,33 @@ class Chapter {
   }
 
   skip(array, n) {
-    const output = array.slice();
-    output.splice(0, n);
-    return output;
+    if (typeof array === 'number') {
```

Comment:
same thing, let's rename this argument, since this line confuses a bit.

Reply:
done